### PR TITLE
Command() parse args correctly

### PR DESCRIPTION
**Pull request by paulbalandan**
**Description**
The `command()` common function presently parses the arguments list by using a call to `explode` only. While this is not a problem when list consists only of arguments, e.g. `test param1 param2`. This may cause problems, however, when we supplied options to the list, e.g. `test param1 -option1 value1 param2`. The `explode` call will give a one-dimensional array where the option keys themselves are the values. So instead of an option `option1 => $value1` we get instead `[1] => option1`.

This PR attempts to emulate `CLI::parseCommandline` in terms of parsing arguments.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Common.php [hunk 1]
```diff
@@ -163,8 +163,31 @@ function command(string $command)
 	{
 		$runner = service('commands');
 
-		$params  = explode(' ', $command);
-		$command = array_shift($params);
+		$args    = explode(' ', $command);
+		$command = array_shift($args);
+
+		$params = [];
+		for ($i = 0, $c = count($args); $i < $c; $i++)
```
Comment (samsonasik) [inline system/Common.php:170]: can use foreach which is faster, ref https://phpsandbox.io/n/super-fog-72g7 

```bash
For took: 0.069ms
For with count() took: 0.442ms
Foreach took: 0.132ms
```

Comment (paulbalandan) [reply system/Common.php:170]: Ok. But how will I check the next item `[$i + 1]` using `foreach`? I tried foreach first but stumbled here.

Comment (samsonasik) [reply system/Common.php:170]: well, you can do: 

```php
foreach ($args as $i => $arg)
{
    if ($args[$i+1] === 'foo') 
   {

   }
}
```

Comment (paulbalandan) [reply system/Common.php:170]: Alright. I forgot that `foreach` has another syntax. Will change.

**Pull request merged**